Hermes-Marketing/hermes
# 6 create get endpoints

## Describe your changes
1. Implemented GET endpoint for getting all records within the businesses collection in firebase
2. defined the Pydantic ORM model Business to handle interacting with the database
3. defined the Business schema to be used as a response model

## Issue ticket number and link

[Issue 6](https://github.com/Hermes-Marketing/hermes/issues/6)

## How to test changes
Navigate to the fast api docs to the business collection
trigger the endpoint with a valid collection name (businesses is the only one in the database)

Observe the response model, and confirm 200 status code.

## Checklist before requesting a review
